ONSdigital/census-rm-case-processor
# Ignore RESPONDENT_AUTHENTICATED messages

# Motivation and Context
We are receiving RESPONDENT_AUTHENTICATED on the same topic as SURVEY_LAUNCHED messages, which we can't deserialise because it's an unexpected event type, although it is in the event dictionary.

# What has changed
Ignore the RESPONDENT_AUTHENTICATED messages.

# How to test?
Publish a RESPONDENT_AUTHENTICATED event and check that Case Processor doesn't blow up.

# Links
Trello: https://trello.com/c/ZhSn40ut
